DirectXTex/DirectXTex.h

```diff
@@ -528,11 +528,9 @@ namespace DirectX
         // levels of '0' indicates a full mipchain, otherwise is generates that number of total levels (including the source base image)
         // Defaults to Fant filtering which is equivalent to a box filter
 
-    HRESULT __cdecl CalculateAlphaCoverage(
-        _In_ const Image& srcImage, _In_ float alphaReference, _In_ float alphaScale, _Out_ float& coverage);
     HRESULT __cdecl ScaleMipMapsAlphaForCoverage(
```

Comment:
Looks good being a single function. I do wonder if the name is obvious about what it does, but I can't think of a better one...